docs/guides/modules/orchestrate/pages/jobs-steps.adoc

```diff
@@ -117,17 +119,18 @@ stateDiagram-v2
    
     pending --> running
 
-    pending --> skipped : Only build PRs setting enabled<br/>ci-skip<br/>No project found<br/>Abusive<br/>Sanctioned
+    pending --> not_run : Only build PRs setting enabled
```

Comment:
Does this transition still exist? I think we now either create the pipeline or not based on this setting, but if we create a job that will be allowed to complete.